catalog/github/webhook.js

```diff
@@ -1,34 +1,64 @@
 var request = require('request');
 
 /**
- *   Feed to create a webhook on Github
- *  @param {object} params - information about the trigger
- *  @param {string} repository - repository to create webhook
- *  @param {string} username - github username
- *  @param {string} accessToken - github access token
- *  @param {string} events - list of the events the webhook should fire on
- *  @return {object} whisk async
+ * Feed to perform webhook operations on Github
+ *
+ * @param {object}
+ *            triggerName - information about the trigger
+ * @param {string}
+ *            repository - repository to create webhook
+ * @param {string}
+ *            username - github username
+ * @param {string}
+ *            accessToken - github access token
+ * @param {string}
+ *            events - list of the events the webhook should fire on
+ * @param {string}
+ *            lifecycleEvent - the operation the feed should perform, currently
+ *            support list, create, delete
+ * @param {string}
+ *            hookid - the id of the webhook to delete
+ * @param {string}
+              endpoint - the host of openwhisk deployment
+ * @return {object} whisk async
  */
 function main(params) {
   var username = params.username;
   var repository = params.repository;
   var accessToken = params.accessToken;
 
-  var endpoint = 'openwhisk.ng.bluemix.net';
-  var lifecycleEvent = params.lifecycleEvent;
-  var triggerName = params.triggerName.split("/");
+	var endpoint = params.endpoint;
+	var lifecycleEvent = params.lifecycleEvent;
+	var triggerName = params.triggerName.split("/");
 
-    // URL of the whisk system. The calls of github will go here.
-  var whiskCallbackUrl = 'https://' + whisk.getAuthKey() + "@" + endpoint + '/api/v1/namespaces/' + triggerName[1] + '/triggers/' + triggerName[2];
+	// URL of the whisk system. The calls of github will go here.
+	var whiskCallbackUrl = 'https://' + whisk.getAuthKey() + "@" + endpoint
+			+ '/api/v1/namespaces/' + triggerName[0] + '/triggers/'
+			+ triggerName[1];
```

Comment:
On using the default value for `triggerName`, that is passed implicitly to this action, the URL would be like this:
'https://' + whisk.getAuthKey() + "@" + endpoint + '/api/v1/namespaces/' + triggerName[**1**] + '/triggers/' + triggerName[**2**]